### ISSUE TITLE
componentWillReceiveProps has been renamed

### ISSUE THREAD

**Issue by arhtudormorar** [YGYOOO/react-draggable-tags#44]
Hi!

  I get the following error:

```
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: e
```

**Comment by YGYOOO** [comment]
Hi, thanks for reporting this. I will fix it maybe several days later.

But actually it's just a warning due to your react version, not an error. Everything about react-draggable-tags should works fine and I think you will not see this waring in the production environment.

**Comment by YGYOOO** [comment]
Hi, it has been fixed in the latest version 1.0.0.
